qabot/env_maintenance.py

```diff
@@ -0,0 +1,165 @@
+            # Run the commands
+            result = subprocess.run(
+                restart_command, capture_output=True, text=True, check=True
+            )
+            log.info(f"Output from command when rolling deployment: {result.stdout}")
+            result = subprocess.run(
+                status_command, capture_output=True, text=True, check=True
+            )
+            log.info(
+                f"Output from command when checking deployment status: {result.stdout}"
+            )
+            return msg
+        except subprocess.CalledProcessError as e:
+            log.info(e.stderr)
+            return f"Failed to roll service(s) on {env_name}, please try again or contact QA team"
+
+    def scaleup_namespace(self, env_name):
+        """
+        Scaleup the given namespace in gen3 environments
+        """
+        log.info(f"Running scaleup-namespace on {env_name}")
+        script_path = "/src/qabot/scripts/scaleup-namespace.sh"
+        try:
+            os.chmod(script_path, 0o755)
+            # Run command to scaleup the namespace
+            command = [
+                script_path,
+                env_name,
+            ]
+            result = subprocess.run(
+                command,
+                capture_output=True,
+                text=True,
+                check=True,
+                input="yes\n",
+            )
+            log.info(f"Output from command when scaling up namespace: {result.stdout}")
+            command = [
+                "kubectl",
+                "rollout",
+                "status",
+                "deployment",
+                "-n",
+                env_name,
+            ]
+            result = subprocess.run(command, capture_output=True, text=True, check=True)
+            log.info(
+                f"Output from command when checking scaleup-namespace status: {result.stdout}"
+            )
+            return f"Namespace {env_name} has been scaled up. :rocket:"
+        except subprocess.CalledProcessError as e:
+            log.info(e.stderr)
+            return f"Failed to scaleup namespace {env_name}, please try again or contact QA team"
+
+    def run_gen3_job(self, job_name, env_name):
+        """
+        Run gen3 job in the given namespace in gen3 environments
+        """
+        log.info(f"Running run-gen3-job on {env_name}")
+        try:
+            command = [
+                "kubectl",
+                "delete",
+                "job",
+                f"{job_name}-qabot",
+                "-n",
+                env_name,
+            ]
+            result = subprocess.run(command, capture_output=True, text=True, check=True)
+            log.info(
+                f"Output from command when deleting existing gen3 job {job_name}: {result.stdout}"
+            )
+            command = [
+                "kubectl",
+                "create",
+                "job",
+                f"--from=cronjob/{job_name}",
```

Comment:
for etl the job name varies, we handled it in gen3-code-vigil, we should do the same here